spinegeneric/cli/generate_figure.py

```diff
@@ -501,16 +522,19 @@ def main():
         metric = file_to_metric[csv_file_small]
 
         # Fetch mean, std, etc. per site
-        results_dict = aggregate_per_site(dict_results, metric)
+        results_dict = aggregate_per_site(dict_results, metric, dict_exclude_subj)
 
         # Make it a pandas structure (easier for manipulations)
         df = pd.DataFrame.from_dict(results_dict, orient='index')
 
         # Compute statistics
-        if metric not in SITES_TO_EXCLUDE.keys():
+        if metric not in dict_exclude_subj.keys():
             sites_to_exclude = []
         else:
-            sites_to_exclude = SITES_TO_EXCLUDE[metric]
+            for subject in dict_exclude_subj[metric]:
```

Comment:
this functionality is not documented in the help, and using a variable called "*_subj" to refer to a site is confusing. I Would add a another variable (and flag) to remove a site.